src/csharp/Grpc.Tools/build/Grpc.Tools.props

```diff
@@ -4,8 +4,9 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
-  <!-- Name of this file must match package ID. -->
-  <!-- Packages will be split later. -->
+  <!-- Name of this file (Grpc.Tools.props) must match package ID (Grpc.Tools). -->
+
+  <!-- TODO(tonydnewell) This could be simplified by combining these files -->
```

Comment:
nit: I'd be more concrete in the comment:
"The files below were originally separate to allow Grpc.Tools and Google.Protobuf.Tools split, but since we abandoned that plan, they logic could now be simplified by combining these files together."